Source/Assimil8or/Assimil8or.cpp

```diff
@@ -631,18 +853,23 @@ void Assimil8orPreset::write (juce::File presetFile)
     write (presetFile, presetProperties.getValueTree ());
 }
 
+void Assimil8orPreset::clearVars()
+{
+    curPresetSection = juce::ValueTree();
+    channelProperties = ChannelProperties();
+    curChannelSection = juce::ValueTree();
+    zoneProperties = ZoneProperties();
+    curZoneSection = juce::ValueTree();
```

Comment:
You can use brace initializers for all of these, to indicate default object of variable type
`curPresetSection = {};`
etc